src/tests/java/picard/vcf/TestFilterVcf.java

```diff
@@ -43,6 +45,31 @@
     private final File INPUT = new File("testdata/picard/vcf/filter/testFiltering.vcf");
     private final File BAD_INPUT = new File("testdata/picard/vcf/filter/testFilteringNoSeqDictionary.vcf");
 
+
+    /* write content of javascript in the returned file */
+	private File quickJavascriptFilter(String content) throws Exception {
+		final File out = File.createTempFile("jsfilter", ".js");
+		out.deleteOnExit();
+		PrintWriter pw = new PrintWriter(out);
+		pw.println(content);
+		pw.flush();
+		pw.close();
+		return out;
```

Comment:
I think you could save a few lines with

```
try(PrintWriter pw = new PrintWriter(out){
  pw.println(content)
}
```

the close and flush should be called implicitly that way.